helm-chart/renku-gateway/values.yaml

```diff
@@ -61,14 +61,18 @@ global:
   ## Specify the information required to connect to a redis instance.
   ## All the values below are required.
   redis:
-    ## Host is optional if a subchart for redis is used, otherwise it needs to be provided below.
-    # host: 
-    ## The label used to enable the network policy that allows the gateway to connect to redis.
-    ## If not defined below, it will default to <redis hostname>-client.
-    # clientLabel:
-    isSentinel:
+    sentinel:
```

Comment:
We're not really consistent on this, but I'm wondering if we should just comment all the global values that we're expecting to be set on the renku chart in a standard deployment? After all, what's set here will have no effect whatsoever.

Reply:
@ableuler sure I can comment all of this out. And this will be consistent across all the other repos that use the global redis.

Reply:
At first I thought sure we can comment out the global values in the component repos. But then the helm chart linting fails. I think it is better to leave it uncommented than to have to specify a bunch of values when you run helm lint.

Reply:
you're right, didn't really think about the linting